config.js

```diff
@@ -46,7 +46,7 @@ var config = {
     },
 
     // BOSH URL. FIXME: use XEP-0156 to discover it.
-    bosh: '//jitsi-meet.example.com/' + subdir + 'http-bind',
+    bosh: 'https://jitsi-meet.example.com/' + subdir + 'http-bind',
```

Comment:
Well because of some other changes the visitors were broken because of the `//` vs `https://` setting. And that was why the first commit exists.  I can move it in another PR.

Reply:
Nah let's have it